ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -136,8 +134,11 @@ trait ParameterStorageBackend {
 }
 
 object ParameterStorageBackend {
-  case class LedgerEnd(lastOffset: Offset, lastEventSeqId: Long)
+  case class LedgerEnd(lastOffset: Offset, lastEventSeqId: Long, lastStringInterningId: Int)
```

Comment:
I like the last, since it also re-assures one how to think about the ledger end (it is a pointer, points to the last element instead to the position where the next element will be). We have many range definition, filtration logic where I found it useful to have that notion clear by name as well.